pacttesting/testing.go

```diff
@@ -366,16 +379,7 @@ func IntegrationTest(pactFilePaths []Pact, testFunc func(), retryOptions ...retr
 }
 
 func StopMockServers() {
-	// keep servers alive by default
-	if os.Getenv("STOP_PACT_SERVERS") != "1" {
-		log.Infof("Leaving pact servers running for future test runs")
-		return
-	}
-	for _, pactServer := range pactServerProcesses {
-		if pactServer != nil {
-			pactClient.StopServer(pactServer)
-		}
-	}
+	// left here for backward compatibility.
```

Comment:
Would it be better to write this comment in godoc style above the function so it can be displayed in the description?

`// Deprecated: StopMockServers left here for backwards compatibility, does not stop anything.`

Reply:
Thanks - fixed